### DISCUSSION TITLE
Didn't get my git hub education package in 6 days

### DISCUSSION THREAD

**Discussion by machtvonjoseph** [community/community#137166] [GitHub Education] [answered]
### Hi everyone,

Hello, 
I had my GitHub Education package approved on Aug 23 and I still haven't gotten it. It says to raise a discussion if it doesn't arrive in three days and maximum 5 days. Any help with this?

**Comment by davevad93** [comment, marked as answer]
Hi @machtvonjoseph ,

It usually takes a few days, but sometimes it can take a little longer, even a couple of weeks; the waiting time may vary depending on different factors, such as the volume of requests.

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @machtvonjoseph was approved and your benefits are now available.`

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
